app/Scripts/boid.gd

```diff
@@ -26,8 +26,10 @@ var boids = []
 
 func _ready():
 	self.set_meta("Boid", false)
-	get_parent().get_node("Enemy").connect("_enemy_moused_over_true", self, "_enemy_moused_over_true")
-	get_parent().get_node("Enemy").connect("_enemy_moused_over_false", self, "_enemy_moused_over_false")
+	#get_parent().get_node("Enemy").connect("_enemy_moused_over_true", self, "_enemy_moused_over_true")
+	#get_parent().get_node("Enemy").connect("_enemy_moused_over_false", self, "_enemy_moused_over_false")
```

Comment:
get_node only gets the first found instance, so I dont think this code was working anyway. just commenting this out for now